src/main/resources/application.yml

```diff
@@ -20,6 +20,8 @@ application:
         - t_,
         - v_
     add_table_name_jdl: true
+    jdl-type-overrides:
+        decimal: FLOAT
```

Comment:
Are you not going to override the type for everyone?
Would it not be better to define override for the specific test only?

If someone runs the code without reading how it works, he will get Float instead of BigDecimal

Reply:
That was just an example or the syntax.  I figured they would have a look and tweak the settings anyway.  I can remove it or change it to some a type that is UNSUPPORTED like JSON_AS_TEXT or YEAR_AS_TEXT.  Either way as I never use the default file because I am only using Postgres.

There is a test that uses

`@TestPropertySource(properties = "application.jdl-type-overrides.numeric=FLOAT")`

https://github.com/Blackdread/sql-to-jdl/pull/148/files/9857a2024bcdeb2b495fd417c4c2997858bda8c7#diff-f856f64c56d2f6298748720b138ccdf6c082391d0052280dd48c238deba049aa

Reply:
We should have the default configuration with the "best" defaults.
Maybe to put an example of non-existent sql type to a JDL type is alright, so it is there just as an example, with a comment in the yml next to it to explain

Reply:
Done